apps/src/templates/Certificate.jsx

```diff
@@ -44,11 +45,27 @@ function Certificate(props) {
       }
     }).done(response => {
       if (response.certificate_sent) {
+        setStudentName(response['name']);
         setPersonalized(true);
       }
     });
   };
 
+  const getCertificate = () => {
```

Comment:
(minor) I think renaming this to `getCertificatePath` might be a bit clearer, but ok to do this in a future PR if Drone already passed on the current change.